src/tutorial/scheduling/graphing.rst

```diff
@@ -13,15 +13,15 @@ The :cylc:conf:`flow.cylc` File Format
 
 .. ifnotslides::
 
-   We refer to a Cylc workflow as a :term:`Cylc suite`. A Cylc suite is a
+   We refer to a Cylc workflow as a :term:`Cylc workflow`. A Cylc workflow is a
    directory containing a :cylc:conf:`flow.cylc` file. This configuration file is where
    we define our workflow. The :cylc:conf:`flow.cylc` file uses a nested `INI`_-based
    format:
 
 .. ifslides::
 
-   * Cylc workflow == Cylc suite
-   * Cylc suite is a directory containing a :cylc:conf:`flow.cylc` file
+   * Cylc workflow == Cylc workflow
```

Comment:
Oh? 😆

Reply:
:rofl:  god, the perils of automatic string-replacement